.travis/setupenv.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# Script to setup data paths
+
+# For Ubuntu 14.04: add pyroot to PYTHON_PATH
+export PYTHONPATH="/usr/share/python-support/root:/usr/lib/python2.7/dist-packages/"
```

Comment:
Let's test on `python 2.6.6` for now.

Reply:
There is no distribution of `pyroot` built against Python 2.6 in Debian, but it seems binary compatible.

Reply:
Again check @jsturdy's implementation to see if this sort of issue is resolved